lib/decidim/votings/vote_confirmation_engine.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "rails"
+require "active_support/all"
+
+require "decidim/core"
+
+module Decidim
+  module Votings
+    # Decidim's <EngineName> Rails Engine.
+    class VoteConfirmationEngine < ::Rails::Engine
+      isolate_namespace Decidim::Votings::VoteConfirmation
+      paths["db/migrate"] = nil
+
+      routes do
+        get "confirm", controller: "confirmations"
```

Comment:
Also, in nVotes server we can configure any URL pattern, I would use something like /confirm/:election_id/:voter_id/ as in the nVotes example, without querystring params.

Reply:
ok